homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js

```diff
@@ -0,0 +1,24 @@
+const a = 24 + 68;
+const b = 37 - 16;
+const c = 100 / 5;
+const x = 4 * 16;
+const y = 4 ** 2;
+const z = 23 % 5;
+
+const add = a + b;
```

Comment:
The common practice is to make a variable, if you like to use it twice at least. 
So, all these vars `add`, `diff`, etc are redundant.
You can just calculate them on the fly:
```js
console.log(`Addition: ${a} + ${b} = ${a + b}`);
```